objects.go

```diff
@@ -2,12 +2,15 @@ package main
 
 import "github.com/faiface/pixel"
 
-// Gravity is vector representing standard gravity force accelleration vector
+// Gravity is vector representing standard gravity force accelleration vector in m*s^{-2}
 var Gravity = pixel.Vec{
```

Comment:
This probably should be `const`, also. But I am not sure if it is possible in go lang.

Reply:
Go does not support constant structs